CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This is a breaking change to the Kubernetes service discovery.
 * [ENHANCEMENT] Add Azure SD metrics.
 * [ENHANCEMENT] Add fuzzy search to `/graph` textarea.
 * [ENHANCEMENT] Always show instance labels on target page.
+* [BUGFIX] Validate query end time is not before start time.
```

Comment:
Perhaps add that negative staleness deltas are now rejected when provided no the command line? (In principle, somebody might have done so, and now the server wouldn't start up anymore.)